### PR TITLE
Standardize electric motors

### PR DESCRIPTION
**Pull request by ArcanErasmus**
#### Summary
```SUMMARY: Balance "Standardize electric motor sizes, components, and outputs"```

#### Purpose of change
Currently, electric motor stats are all over the place. For instance, the regular electric motor is far more massive than the large or enhanced motors. Secondly, stats don't seem to line up very well with real-world equivalents (though those vary considerably).

In addition, motor efficiencies are a little bit too standard. Generally speaking, the larger the motor is, the more efficient it will be. I wanted this to be modeled in-game, which would also give a boost to the value of the larger, uncraftable, motors.

Lastly, the use of power and drain values calculated based on converted real-life values meant that the exact draw and output would be hard to eyeball for a player.

#### Describe the solution
First of all, motor volumes and masses were changed to be in the correct order (i.e. a more powerful motor shouldn't be smaller than a weaker motor). Second, since all motors are basically just fat chunks of metal, and thus have a pretty consistent mass, the mass of each motor was changed to be the same relative to its volume.

Next, drain was changed to nice round numbers, and power calculated based on the drain, multiplied by an increasing efficiency (as in, larger motors are more efficient). This should make it easier for a player to eyeball how much power they need to supply their vehicle with.

At this point, the only items that really received significant balance change (besides re-ordering mass/volume values) are the tiny, and enhanced and super motors. the tiny motor was nearly as heavy as the small motor, despite having one tenth of the volume and power output. As for the largest two motors, they didn't feel impactful enough, so the outputs have been tweaked. The enhanced motor has been changed to be about twice as powerful as the large motor, making it slightly weaker, but the super electric motor has been boosted by nearly 20%. It seemed quite odd that the electric sports car would end up being more powerful than the electric racecar, and that the super electric motor was not a significant step up. Given its rarity (and given that it won't have much impact on gameplay overall), I think this modest bump is logical.

Lastly, some work was done to standardize crafting/uncrafting components. Given that the tiny motor was the only one to contain a plastic chunk, I decided to remove that entirely. Beyond that, the amount of metal pieces contained in each motor was also out of order, so this was realigned to the respective sizes. In addition, all uncraftable motors were changed to include some amount of electronic scrap.

#### Describe alternatives you've considered
It might be easier on the player to make the power output numbers nice and round, instead of drain. But since you can only choose one (unless you radically change the efficiencies), I went with the one that keeps values closer to what they were before.

### CODE REVIEW

**Comment by Fris0uman** [comment]
See https://github.com/CleverRaven/Cataclysm-DDA/pull/27978 for rational behind weight volume powet and stuff

**Comment by ArcanErasmus** [comment]
Since this is my first PR for this project, I am not sure if folks are waiting on me to do something before it can merge. Is there anything I am missing?

**Comment by Fris0uman** [comment]
Merging takes time, if you already took care of the review you got you just have to wait now.

File: data/json/vehicleparts/motor.json [hunk 1]
```diff
@@ -85,14 +82,16 @@
     "item": "motor_large",
     "difficulty": 3,
     "durability": 400,
-    "power": 149200,
-    "energy_consumption": 166000,
+    "power": 138000,
```
Comment (mlangsdorf) [inline data/json/vehicleparts/motor.json]: Please make this 150,000 W of power at 165,000 W of consumption.

File: data/json/vehicleparts/motor.json [hunk 2]
```diff
@@ -43,14 +39,13 @@
     "item": "motor_small",
     "difficulty": 1,
     "durability": 120,
-    "power": 7460,
-    "energy_consumption": 8500,
+    "power": 7040,
```
Comment (mlangsdorf) [inline data/json/vehicleparts/motor.json]: should be 7,500 W at 10,444 W consumption, making it the 7.5 kW generator running in reverse.

File: data/json/vehicleparts/motor.json [hunk 3]
```diff
@@ -23,15 +23,11 @@
     "looks_like": "motor_small",
     "difficulty": 1,
     "durability": 80,
-    "power": 750,
-    "energy_consumption": 854,
+    "power": 800,
```
Comment (mlangsdorf) [inline data/json/vehicleparts/motor.json]: should be 780 W at 1,492 W consumption, the same as the car alternator running in reverse.

Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: That is 52% efficiency, which is extremely low for an electric motor!

File: data/json/vehicleparts/motor.json (hunk 2 above)
Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: 72% efficiency is also quite low for an electric motor. While that would make sense for a generator, it really wouldn't for a motor.

File: data/json/vehicleparts/motor.json (hunk 3 above)
Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: Yeah, further research leads me to believe that alternators are **far** less efficient at generating power than electric motors are at turning that power into motion.

File: data/json/vehicleparts/motor.json (hunk 2 above)
Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: Same as alternator comment. Generators have lower efficiency than motors.

File: data/json/vehicleparts/motor.json (hunk 1 above)
Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: I can update one or both of the numbers here, but I would like to keep the theme of efficiency rising as the motor sizes rise. This is true of real electric motors (at least on average). The regular motor has an efficiency of 90%, and the enhanced has 94%, so I would like to keep the large at or around 92%.

Comment (ArcanErasmus) [reply data/json/vehicleparts/motor.json]: To keep that efficiency, I could do 150k/163k, or 151.8k/165k. Do you have a preference?

Comment (mlangsdorf) [reply data/json/vehicleparts/motor.json]: 150K, 163K.

**Comment by mlangsdorf** [comment]
looks like you'll need to run and update the vehicle drag and possibly the vehicle efficiency tests.

**Comment by ArcanErasmus** [comment]
Yeah, I had to leave in a hurry, but I figured I would push the tweak and let the tests run while I was out.

**Pull request merged**